package.json

```diff
@@ -5,16 +5,15 @@
     "production": "cross-env NODE_ENV=production postcss public/tailwind.css -o public/build/tailwind.css",
     "start": "npm run serve"
   },
-  "dependencies": {
-    "autoprefixer": "^9.5.1",
-    "tailwindcss": "^1.0"
-  },
+  "dependencies": {},
   "devDependencies": {
     "@fullhuman/postcss-purgecss": "^1.2.0",
-    "concurrently": "^4.1.0",
+    "autoprefixer": "^9.8.5",
+    "concurrently": "^5.2.0",
     "cross-env": "^5.2.0",
     "cssnano": "^4.1.10",
     "live-server": "^1.2.1",
-    "postcss-cli": "^6.1.2"
+    "postcss-cli": "^7.1.1",
+    "tailwindcss": "^1.5.2"
```

Comment:
Since Tailwind is included in the final build, I think it would be wise to move it to the `dependencies`, not `devDependencies`